python-lib/config.py

```diff
@@ -82,27 +82,38 @@ def get_hdfs_location(dataset_config: Mapping[AnyStr, Any], sf_stage_name: AnySt
     return f'{sf_stage_name}{path}/'
 
 
-def get_snowflake_to_hdfs_query(sf_location: AnyStr, sf_table_name: AnyStr) -> AnyStr:
+def get_snowflake_to_hdfs_query(sf_location: AnyStr, sf_table_name: AnyStr,
+                                sf_schema: List[Mapping[AnyStr, AnyStr]]) -> AnyStr:
     """
     Gets a COPY statement for copying from a Snowflake Table to an HDFS location
-    :param sf_location:
-    :param sf_table_name:
+    :param sf_location: HDFS location
+    :param sf_table_name: Snowflake table
+    :param sf_schema: Snowflake schema
     :return: a SQL COPY statement
     """
     # moving this function here isn't strictly necessary as it's not re-used, but it makes
     # things significantly easier to unit test.
 
+    # Generate SELECT clause and cast TIMESTAMP_TZ, TIMESTAMP_LTZ to TIMESTAMP
+    # This is required as COPY command does not support TZ and LTZ. See AOBS-491 for more details
```

Comment:
```suggestion
    # This is required as COPY command does not support TZ and LTZ
```

This is a public repo so references to Jira tickets aren't helpful.

I think it would be helpful to add a bit to the README that describes this limitation.